File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/kotlin/mouredev.kt

fun main() {
    fizzbuzz()
}

private fun fizzbuzz() {

    for(number in 1..100) {

        if (number % 3 == 0 && number % 5 == 0) {
            println("fizzbuzz")
        } else if (number % 3 == 0) {
            println("fizz")
        } else if (number % 5 == 0) {
            println("buzz")
        } else {
            println(number)
        }
    }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/swift/mouredev.swift
import Foundation

func fizzbuzz() {

    for number in 1...100 {

        if number % 3 == 0 && number % 5 == 0 {
            print("fizzbuzz")
        } else if number % 3 == 0 {
            print("fizz")
        } else if number % 5 == 0 {
            print("buzz")
        } else {
            print(number)
        }
    }
}

fizzbuzz()